src/Analyzers/CSharp/CodeFixes/UseCollectionExpression/CSharpCollectionExpressionRewriter.cs

```diff
@@ -350,32 +351,41 @@ void AddCommaIfMissing()
         }
 
         static CollectionElementSyntax CreateCollectionElement(
-            CollectionExpressionMatch<TMatchNode>? match, ExpressionSyntax expression)
+            bool useSpread, ExpressionSyntax expression)
         {
-            return match?.UseSpread is true
+            return useSpread
                 ? SpreadElement(
                     Token(SyntaxKind.DotDotToken).WithLeadingTrivia(expression.GetLeadingTrivia()).WithTrailingTrivia(Space),
                     expression.WithoutLeadingTrivia())
                 : ExpressionElement(expression);
         }
 
-        CollectionElementSyntax CreateElement(
+        IEnumerable<CollectionElementSyntax> CreateElements(
             CollectionExpressionMatch<TMatchNode> match, string? preferredIndentation)
         {
             var node = match.Node;
 
             if (node is ExpressionStatementSyntax expressionStatement)
             {
-                // Create: `x` for `collection.Add(x)` or `.. x` for `collection.AddRange(x)`
-                return CreateCollectionElement(
-                    match,
-                    ConvertExpression(expressionStatement.Expression, expr => IndentExpression(expressionStatement, expr, preferredIndentation)));
+                // Create:
+                //
+                //      `x` for `collection.Add(x)`
+                //      `.. x` for `collection.AddRange(x)`
+                //      `x, y, z` for `collection.AddRange(x, y, z)`
```

Comment:
this last case is why this method had to start being able to return *multiple* elements.  Up until this case we didnt' have any AddRange methods taht could take `params T[]`.  However, the immutable-builder apis allow this, so this logic here was tweaked.